j5/boards/base.py

```diff
@@ -11,6 +11,8 @@
 class Board(metaclass=ABCMeta):
     """A collection of hardware that has an implementation."""
 
+    BOARDS: List['Board'] = []  # All the boards.
```

Comment:
Please can we have a comment to explain what the purpose of this list is?

e.g. "This list contains every instance of `Board` that has had its `setup` method called, and is used by `BaseRobot.make_all_safe` when Python exits to ensure every board is made safe."